_docs/style/text-conventions/alert-messages.md

```diff
@@ -0,0 +1,291 @@
+
+Write brief and simple sentences that clearly state the problem. Separate the sentences with a period (or question mark, if applicable), but not with a semicolon.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| Provide a name for each domain. *null* isn't a valid domain name. | You must provide a name for each domain and *null* isn't a valid name. |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+Avoid using only uppercase letters.
+
+Lines with excessive capitalization are hard to read. Use all uppercase letters only for words that require it, such as a keyword, a data type, or a specific table name that's displayed in all uppercase letters to a database user. See [Capitalization]({{site.baseURL}}/style/capitalization.html) for more information.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| The requested image $UUID has automatic disk resizing disabled. | THE REQUESTED IMAGE $UUID HAS AUTOMATIC DISK RESIZING DISABLED. |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+When possible, include a recommendation, either a potential fix or a reference to a document for more information.
+
+Messages should provide specific information about how the user should continue.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| The system is out of virtual IP addresses. Contact Support so they can allocate more virtual IP addresses. <br/> <br/>The value -1.0 can't be accepted. Specify a positive integer value for the volume size. <br/> <br/> | The system is out of virtual IP addresses. <br/> <br/>The value -1.0 can't be accepted. |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+Be specific and provide as much detailed information as possible.
```

Comment:
this use/ don't use column header shouldn't wrap. we have GOT to fix this column auto spacing thing. 
is there anything we can do before we push this content?

Reply:
I added line breaks and that resolved it.